CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,8 @@
+### Breaking
+
+- [engine] Engine always encrypts secret return values in Invoke
+  [#10239](https://github.com/pulumi/pulumi/pull/10239)
```

Comment:
I feel like this may not be clear enough for end users. Given the scary "Breaking" header, it would be nice to explain what exactly is broken.

Reply:
I thought so too so I wrote and merged a second PR to fix my changelog entry. https://github.com/pulumi/pulumi/pull/10252